python/afdko/autohint.py

```diff
@@ -1237,7 +1237,7 @@ def hintFile(options):
 		else:
 			if os.path.exists(tempBezNew):
 				os.remove(tempBezNew)
-			command = AUTOHINTEXE + " %s%s%s%s -s %s -f \"%s\" \"%s\"" % (verboseArg, suppressEditArg, supressHintSubArg, decimalArg, NEWBEZ_SUFFIX, tempFI, tempBez)
+			command = '"%s" %s%s%s%s -s %s -f "%s" "%s"' % (AUTOHINTEXE, verboseArg, suppressEditArg, supressHintSubArg, decimalArg, NEWBEZ_SUFFIX, tempFI, tempBez)
```

Comment:
if you called the subprocess with a list of strings instead a single string, then subprocess would take care of the quoting for you

Reply:
`autohint.py` is on the chopping block

Reply:
and `fdkutils.runShellCmd` is used in many places; any changes to it need to be done carefully.